Xamarin.Forms.Platform.Android/ResourceManager.cs

```diff
@@ -114,17 +114,21 @@ public static int GetStyleByName(string name)
 
 		public static void Init(Assembly masterAssembly)
 		{
-			DrawableClass = masterAssembly.GetTypes().FirstOrDefault(x => x.Name == "Drawable" || x.Name == "Resource_Drawable");
+			DrawableClasses = masterAssembly.GetTypes().Where(x => x.Name == "Drawable" || x.Name == "Mipmap").ToArray();
```

Comment:
- I'm wondering if this should be setup to ensure that Drawable is the first element of the array. If mipmap is first that's going to cause an extra search for most scenarios

- also I'm thinking GetTypes should either be stored locally or looped over to set all of these types up. Each call to GetTypes probably causes some overhead

Reply:
To `Drawable` was always the first element you can sort the enumeration by name `.OrderBy(x => x.Name)`.

Reply:
My concern there is getting to LINQ'y in code that's going to most likely run during startup 
So trying to optimize this code as much as possible would be ideal

Reply:
I think we can remove LINQ here and add lazy initialization. As well as to cache resource ID, so that each time not to iterate through all the properties.

Reply:
Yea that looks like what I was envisioning on the Init part which I think is good enough for this PR

The IdHash part I'm wondering about. I would just be scared of that growing too much. 
I've been wanting to look at using an LruCache (https://developer.android.com/reference/android/util/LruCache) for storing bitmaps/drawables in these scenarios to improve with performance and to stay mindful of not creating an unbounded data structure. But I think that part can be a discussion for a different PR